web/kratos/Dockerfile

```diff
@@ -7,6 +7,7 @@ RUN apk add gettext
 COPY . /usr/bin/app/kratos
 
 RUN mkdir -p /etc/config/kratos && chmod a+x /usr/bin/app/kratos/entrypoint.sh && \
+    chmod a+x /usr/bin/app/kratos/oidc-setup.sh && \
```

Comment:
The `a` is needed?

Reply:
Copied from the line above so they are the same.